[W11-3] TravelBuddy

chung-ming opened the pull request.

kylase-learning commented:
## Feedback (Week 7; v1.1)
 
### Github Project Page
- [x] Update Project description
- [x] Update Link to Project Page
 
### README
- [x] UI Mockup updated in project website
- [ ] Make the project website look like a product home page (aka remove the top nav bar)
- [x] AB4 acknowledgement
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's
 
### [Project Page](https://cs2103-ay1819s2-w11-4.github.io/main/)
- [x] Update to remove AB4 reference
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's

### About Us Page
- [x] Updated with photos
- [x] Roles and components assigned

UG/DG feedback will be provided to you in the documents.

Generally, I can see that all of you do not have huge issue with the workflow. Keep up the good work! 👍

kylase-learning commented on src/main/java/seedu/address/logic/commands/AddCommand.java:
```diff
@@ -2,60 +2,64 @@
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New place added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This place already exists in the address book";
```
The constant name should be `MESSAGE_DUPLICATE_PLACE`? 

Let's try to fix all of these 😄

chung-ming replied:
Okay

kylase-learning commented on src/main/java/seedu/address/logic/commands/DeleteCommand.java:
```diff
@@ -9,21 +9,21 @@
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Place: %1$s";
```
Should be `MESSAGE_DELETE_PLACE_SUCCESS`?

kylase-learning commented on src/main/java/seedu/address/logic/commands/EditCommand.java:
```diff
@@ -20,88 +21,93 @@
+            + PREFIX_DESCRIPTION + "No description";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Place: %1$s";
```
Should be `MESSAGE_EDIT_PLACE_SUCCESS`?

chung-ming committed:
Merge pull request #20 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

chung-ming committed:
Fix bug ReadMe url not working

chung-ming committed:
Implement JUnit Testing for Chart feature

chung-ming committed:
Fix failed tests by adding newline to EOF

nivenang committed:
Fix bug where calling select, generate, search, generate in order causes RightParentPanel to get stuck on null ExpandedPlacePanel

nivenang committed:
Merge pull request #117 from chung-ming/chart-testing

Implement JUnit Testing for Chart feature

nivenang committed:
Fix checkstyle errors

PremBagda committed:
Merge pull request #118 from nivenang/master

Fix RightParentPanel and null ExpandedPlacePanel bug

chung-ming committed:
Implement JUnit Testing for Chart feature

nivenang committed:
Merge pull request #119 from chung-ming/chart-testing

Implement JUnit Testing for Chart feature

nivenang committed:
Merge pull request #25 from CS2103-AY1819S2-W11-3/master

Merge pull request

nivenang committed:
Add functionality to display search query upon successful search

nivenang committed:
Fix checkstyle errors

nivenang committed:
Update UserGuide and graphics for SearchCommand

chung-ming committed:
Update Developer Guide

chung-ming committed:
Merge pull request #121 from nivenang/master

Add functionality to display search query upon successful search

PremBagda committed:
Merge pull request #122 from chung-ming/chart-testing

Update Developer Guide

PremBagda committed:
Fixed unnecessary spaces in UG

chung-ming committed:
Merge pull request #123 from PremBagda/fix_extra_spaces_ug

Fixed unnecessary spaces in UG

chung-ming committed:
Merge pull request #21 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

chung-ming committed:
Documentation housekeeping

PremBagda committed:
Fix PlaceCard and ExpandedPlacePanel alignment issues

Align name of place with rest of the information in PlaceCard.
Fix alignement issues of information displayed in ExpandedPlace Panel.

PremBagda committed:
Merge pull request #124 from chung-ming/modify-docs

 Documentation housekeeping

nivenang committed:
Merge pull request #125 from PremBagda/fix_PlaceHolder_extra_spaces

Fix PlaceCard and ExpandedPlacePanel alignment issues

nivenang committed:
Merge pull request #26 from CS2103-AY1819S2-W11-3/master

nivenang committed:
Fix documentation errors found in PE dry run

PremBagda committed:
Merge pull request #135 from nivenang/master

Fix documentation errors found in PE dry run

chung-ming committed:
Merge pull request #22 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

nivenang committed:
Fix DeleteMultipleCommand bugs found during PE dry run; Implement DeleteMultipleCommandSystemTest

PremBagda committed:
Merge pull request #161 from nivenang/master

Fix DeleteMultipleCommand bugs found during PE dry run

PremBagda committed:
Fix bug for Issue #159

When the same prefix is typed more than once, TravelBuddy only accepts the last prefix. As this is a normal functionality, the User Guide has been updated to reflect this.

nivenang committed:
Update DeveloperGuide search command

nivenang committed:
Update PPP

shaungohsc committed:
Add a initImg method that loads a placeholder image into ExpandedPlacePanel

chung-ming committed:
Merge pull request #162 from nivenang/master

Update DeveloperGuide search command

chung-ming committed:
Update User Guide, Developer Guide and PPP documentation

chung-ming committed:
Implement command shortcuts

chung-ming committed:
Fix failed test cases

nivenang committed:
Merge pull request #163 from chung-ming/modify-docs-2

 Update User Guide, Developer Guide and PPP documentation and implement command shortcuts

shaungohsc committed:
Add Photo class, change Place class to include Photo, updated the ExpandedPlacePanel to allow for scrolling

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main into photo-feature

shaungohsc committed:
Add Photo data field, Change all relevant seedu.travel classes

shaungohsc committed:
[Broken] Incomplete fixes to AddCommandParser, Photo and JsonAdaptedPlace and JsonAdaptedPlaceTest

shaungohsc committed:
[Broken] Change JsonAdaptedPlaceTest and Add PhotoTest

chung-ming committed:
Update documentation to include generate command as undoable. Refer to PR #153.

chung-ming committed:
Update bar chart to increase readability. Display the number of places visited on the top of the bar chart. PR #157

nivenang committed:
Merge pull request #164 from chung-ming/modify-docs-2

Increase readability for bar chart

chung-ming committed:
Modify User Guide

chung-ming committed:
Fix failing test by removing trailing whitespace

nivenang committed:
Merge pull request #165 from chung-ming/modify-docs-2

Modify User Guide

shaungohsc committed:
[Broken] Fix EditCommand and EditComamndParser, photos now display

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main into photo-feature

shaungohsc committed:
Made UI changes to ExpandedPlacePanel, Changed default image

chung-ming committed:
Refactor Display Card to Chart Card, and DisplayListPanel to ChartListPanel.

chung-ming committed:
Refactor DisplayListCard to ChartListCard and DisplayListPanel to ChartListPanel

chung-ming committed:
Update documentation for UG, DG and PPP.

chung-ming committed:
Update DG

PremBagda committed:
Merge pull request #166 from chung-ming/modify-docs-2

Update documentation for UG, DG and PPP

PremBagda committed:
Fix all pe-dry bugs

Fix issues #156, #147, #146 & #127.

chung-ming committed:
Merge pull request #23 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

PremBagda committed:
Fix trailing whitespace issue

PremBagda committed:
Merge branch 'master' into fix_pe-dry_run_bug

chung-ming committed:
Merge pull request #167 from PremBagda/fix_pe-dry_run_bug

Fix pe dry run bugs

PremBagda committed:
Fix DG Trailing Spaces Error.

PremBagda committed:
Fix #1 Load Sample Date issue

travelBuddy_dataFileDoesNotExist_loadSampleData

chung-ming committed:
Merge pull request #168 from PremBagda/fix_cc_help

Fix DG Trailing Spaces Error.

chung-ming committed:
Merge pull request #24 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

shaungohsc committed:
Fix tests and style, fixes for default photo handling

chung-ming committed:
Update DG and PPP

chung-ming committed:
Merge pull request #169 from shaungohsc/photo-feature

Implement Photo Feature

chung-ming committed:
Fix trailing whitespaces in DG

chung-ming committed:
Fix trailing whitespaces in DG

nivenang committed:
Merge pull request #170 from chung-ming/modify-docs

Modify DG and PPP

nivenang committed:
Merge pull request #27 from CS2103-AY1819S2-W11-3/master

chung-ming committed:
Modify PPP and DG

chung-ming committed:
Modify UG, DG and PPP

chung-ming committed:
Fix trailing whitespaces

PremBagda committed:
Merge pull request #171 from chung-ming/modify-docs

Modify PPP and DG

nivenang committed:
Merge pull request #28 from CS2103-AY1819S2-W11-3/master

merge pr

chung-ming committed:
Merge pull request #25 from CS2103-AY1819S2-W11-3/master

Merge changes from team repo

nivenang committed:
Update UserGuide and DeveloperGuide

chung-ming committed:
Merge pull request #174 from nivenang/master

Update UserGuide and DeveloperGuide

chung-ming committed:
Merge pull request #26 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

chung-ming committed:
Update DATE_VISITED parameter in sample data and unit test case

chung-ming committed:
Fixed grammatical error in DG and added RepoSense link to PPP

chung-ming committed:
Modify PPP

shaungohsc committed:
Update UG for Photo feature, Fix command error for Photo FILE_PATH.

shaungohsc committed:
Update default place image to a light grey bar

nivenang committed:
Merge pull request #175 from chung-ming/master

 Update DATE_VISITED parameter in sample data and unit test case

nivenang committed:
Merge pull request #176 from chung-ming/modify-docs-2

Fixed grammatical error in DG and added RepoSense link to PPP

nivenang committed:
Merge pull request #29 from CS2103-AY1819S2-W11-3/master

merge pr

nivenang committed:
Update PPP

nivenang committed:
Fix PPP formatting

nivenang committed:
Fix formatting issues for PPP

shaungohsc committed:
Add additional parsePhoto utility to take care of leading and trailing double quotes around filepaths

shaungohsc committed:
Update UG and remove Photo debug statements

shaungohsc committed:
Update UG for intro language

nivenang committed:
Merge pull request #178 from shaungohsc/master

Updated UG for Photo feature and made QOL changes for "FILE_PATH"

shaungohsc committed:
Merge pull request #177 from nivenang/master

Update PPP for nivenang

nivenang committed:
Update JsonChartBookStorageTest

nivenang committed:
Fix PPP html formatting bug

nivenang committed:
Merge pull request #30 from CS2103-AY1819S2-W11-3/master

Merge pr

nivenang committed:
Add tests for Photo feature to improve coverage

chung-ming committed:
Merge pull request #179 from nivenang/master

Add JUnit tests for Photo feature and JsonChartBookStorageTest to improve coverage

chung-ming committed:
Standardize asciidoc attributes in UG, DG and my PPP.

chung-ming committed:
Remove unused methods to increase coverage

PremBagda committed:
Update PPP & List of Country Codes

Updated PPP to v1.4. Country Codes file has been made more user friendly.

chung-ming committed:
Merge branch 'master' into modify-docs-2

chung-ming committed:
Fixed trailing whitespaces

chung-ming committed:
Merge branch 'modify-docs-2' of https://github.com/chung-ming/main into modify-docs-2

chung-ming committed:
Merge pull request #180 from PremBagda/update_PPP_UG_DG_v1.4

Update PPP & List of Country Codes

PremBagda committed:
Merge pull request #181 from chung-ming/modify-docs-2

Standardize asciidoc attributes and remove unused methods to increase coverage

chung-ming committed:
Merge pull request #27 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

shaungohsc committed:
Commit changes not made by shaungohsc due to merge conflict when pulling

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

PremBagda committed:
Add Country Code Link to UG

PremBagda committed:
Fix newline error

PremBagda committed:
Fix newline error #2

nivenang committed:
Merge pull request #31 from CS2103-AY1819S2-W11-3/master

Merge pr

PremBagda committed:
Fix newline error #3

chung-ming committed:
Add tests for ChartBookStorage

chung-ming committed:
Modify PPP

PremBagda committed:
Merge pull request #183 from chung-ming/master

 Add tests for ChartBookStorage

chung-ming committed:
Merge pull request #182 from PremBagda/update_UG_DG

Add Country Code Link to UG

shaungohsc committed:
Updated UG and PPP

shaungohsc committed:
To prevent merge conflict

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

# Conflicts:
#	docs/UserGuide.adoc

chung-ming committed:
Merge pull request #184 from shaungohsc/master

Update PPP and UG 4.3.1 and 4.5.1

chung-ming committed:
Merge pull request #28 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

chung-ming committed:
Modify docs

PremBagda committed:
Update UG, DG, PPP & CountryCodes file.

chung-ming committed:
Merge branch 'master' of https://github.com/chung-ming/main

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc

shaungohsc committed:
Change Menu CSS to better fit Light-Theme colour-scheme

chung-ming committed:
Merge pull request #185 from PremBagda/update_UG_DG

Update UG, DG, PPP & CountryCodes file.

shaungohsc committed:
Change ExpandedPlacePanel background color

chung-ming committed:
Fix bug in rating chart

nivenang committed:
Merge pull request #32 from CS2103-AY1819S2-W11-3/master

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

nivenang committed:
Update UserGuide, DeveloperGuide, PPP and diagrams for search

shaungohsc committed:
Fix adoc checkstyle and java checkstyle errors

shaungohsc committed:
Fix CSS code quality

shaungohsc committed:
Merge pull request #187 from nivenang/master

 Update UserGuide, DeveloperGuide, PPP and diagrams for search

nivenang committed:
Merge pull request #186 from shaungohsc/master

[UI] CSS changes for LightTheme

shaungohsc committed:
Add ExpandedPlacePanelTest

chung-ming committed:
Modify UG, DG and PPP for Chart feature

chung-ming committed:
Fix checkstyle errors

shaungohsc committed:
Remove unused method from RightParentPanel

shaungohsc committed:
Improve test coverage

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

nivenang committed:
Merge pull request #189 from shaungohsc/master

Increase Test coverage for ExpandedPlacePanel, Place classes

nivenang committed:
Merge pull request #188 from chung-ming/modify-docs

 Modify UG, DG and PPP for Chart feature

shaungohsc committed:
Updated model Diagrams

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

PremBagda committed:
Modify UG & PPP for v1.4

PremBagda committed:
Merge branch 'master' into up_PPP_UG_DG

PremBagda committed:
Fix EOF error.

PremBagda committed:
Merge remote-tracking branch 'origin/up_PPP_UG_DG' into up_PPP_UG_DG

shaungohsc committed:
Merge pull request #190 from PremBagda/up_PPP_UG_DG

Modify UG & PPP for v1.4

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

chung-ming committed:
Merge pull request #29 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

nivenang committed:
Update PPP

chung-ming committed:
Modify PPP

chung-ming committed:
Added non-functional requirements to DG

PremBagda committed:
Update UserGuide, DeveloperGuide & PPP.

PremBagda committed:
Updated RepoSense Date

PremBagda committed:
Merge pull request #191 from nivenang/master

Update PPP

chung-ming committed:
Remove RepoSense until param

nivenang committed:
Merge pull request #193 from PremBagda/UG_&_DG_Update

Update UserGuide, DeveloperGuide & PPP.

nivenang committed:
Merge pull request #192 from chung-ming/modify-dcos-2

 Add Non-Functional Requirements to DG

shaungohsc committed:
Update DG for Photo feature and others

shaungohsc committed:
Update PPP

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

nivenang committed:
Merge pull request #194 from shaungohsc/master

Update PPP DG UG and diagrams

chung-ming committed:
Fix grammer issue in UG

nivenang committed:
Merge pull request #195 from chung-ming/modify-dcos-2

Fix grammer issue in UG

nivenang committed:
Merge pull request #33 from CS2103-AY1819S2-W11-3/master

shaungohsc committed:
Update UG and PPP

PremBagda committed:
Last few fixes.

nivenang committed:
Update diagrams for UG and DG

nivenang committed:
Merge pull request #196 from PremBagda/really_last_lap

Last few fixes.

nivenang committed:
Update UndoRedoSequenceDiagram

nivenang committed:
Update SearchCommandSequenceDiagram and DG

shaungohsc committed:
Update all that stuff, DG UG, diagrams

shaungohsc committed:
Merge branch 'master' into master

nivenang committed:
Update DG

chung-ming committed:
Modify UG, DG and PPP

PremBagda committed:
Merge pull request #198 from shaungohsc/master

Update UG DG and diagrams

PremBagda committed:
Last fix for DG v1.4

PremBagda committed:
Merge branch 'master' into really_last_lap

nivenang committed:
Update DG

nivenang committed:
Merge branch 'master' into master

nivenang committed:
Update DeveloperGuide.adoc

nivenang committed:
Update DeveloperGuide.adoc

nivenang committed:
Merge pull request #199 from chung-ming/modify-dcos-2

Modify UG, DG and PPP

chung-ming committed:
Merge pull request #200 from PremBagda/really_last_lap

Last fix for DG v1.4

shaungohsc committed:
Merge pull request #197 from nivenang/master

Updated diagrams for UG and DG

nivenang committed:
Merge pull request #34 from CS2103-AY1819S2-W11-3/master

merge pr

nivenang committed:
Update PPP

PremBagda committed:
Update DG

PremBagda committed:
Update PPP

PremBagda committed:
Merge pull request #201 from nivenang/master

Update PPP

nivenang committed:
Merge pull request #202 from PremBagda/end

Update DG

nivenang committed:
Merge pull request #35 from CS2103-AY1819S2-W11-3/master

Merge pr

chung-ming committed:
Merge pull request #30 from CS2103-AY1819S2-W11-3/master

Merge latest changes from team repo

chung-ming committed:
Modify PPP

chung-ming committed:
Merge branch 'master' of https://github.com/chung-ming/main

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

# Conflicts:
#	docs/DeveloperGuide.adoc

shaungohsc committed:
Update PPP

shaungohsc committed:
Merge pull request #203 from chung-ming/master

Modify PPP

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

# Conflicts:
#	docs/DeveloperGuide.adoc

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

shaungohsc committed:
Update PPP

chung-ming committed:
Merge pull request #204 from shaungohsc/master

Update PPP

PremBagda committed:
Update for submission

shaungohsc committed:
Merge pull request #205 from PremBagda/end1

Update for submission

chung-ming committed:
Modify PPP

chung-ming committed:
Modify PPP

shaungohsc committed:
Update PPP and add PPP pdf

PremBagda committed:
Merge pull request #206 from chung-ming/modify-docs-3

Modify PPP

shaungohsc committed:
Update PPP

shaungohsc committed:
Change UG tags

shaungohsc committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w11-3/main

PremBagda committed:
Merge pull request #207 from shaungohsc/master

Update PPP

PremBagda committed:
Update numbering

shaungohsc committed:
Merge pull request #208 from PremBagda/thisEnd

Update numbering

nivenang committed:
Merge pull request #36 from CS2103-AY1819S2-W11-3/master

merge pr

nivenang committed:
Update diagram extensions

shaungohsc committed:
Merge pull request #209 from nivenang/master

Update diagram extensions

shaungohsc committed:
Remove

shaungohsc committed:
Remove png

PremBagda committed:
Merge pull request #210 from shaungohsc/master

remove .PNG files

nivenang committed:
Merge pull request #37 from CS2103-AY1819S2-W11-3/master

merge pr

shaungohsc committed:
Change photo resolution

PremBagda committed:
Merge pull request #211 from shaungohsc/master

Change photo resolution

nivenang committed:
Merge pull request #38 from CS2103-AY1819S2-W11-3/master

merge pr

nivenang committed:
Update diagrams for photos

shaungohsc committed:
Merge pull request #212 from nivenang/master

Update photos for UG and DG

PremBagda committed:
Fix for photo issue.

shaungohsc committed:
Merge pull request #213 from PremBagda/3

Fix for photo issue.

nivenang committed:
Update diagrams

shaungohsc committed:
Merge pull request #214 from nivenang/master

Update diagrams

shaungohsc committed:
Final PPP version

nivenang committed:
Update PPP

nivenang committed:
Update diagrams

nivenang committed:
Update PPP

nivenang committed:
Merge pull request #215 from shaungohsc/master

PPP Update

nivenang committed:
Fix ParserUtil bug

PremBagda committed:
Merge pull request #216 from nivenang/master

Update PPP

PremBagda committed:
Final Fix

PremBagda committed:
Merge pull request #217 from PremBagda/Final

Final Fix
